src/platform/platform_linux.cpp

```diff
@@ -154,6 +154,12 @@ bool mp::platform::link(const char* target, const char* link)
 
 bool mp::platform::is_alias_supported(const std::string& alias, const std::string& remote)
 {
+    auto driver = utils::get_driver_str();
+
+    // snapcraft:core{18,20} images don't work on LXD yet
```

Comment:
What is the status of these images working with LXD and why don't they work?

Reply:
They don't have working UEFI booting. It's on their radar. I know this somewhat regresses the LXD backend, but the alternative is to keep `minimal` images for LXD, and use `buildd` for others… I'd rather not.

I was also wondering whether we should disallow the `snapcraft:` remote on LXD altogether.

Reply:
Ok, this was all taken care of on chat.